File: BeeCrowd C++/1009.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main(){
  string nome;
  double salario_fixo,total_vendas;

  cin >> nome;
  cin >> salario_fixo;
  cin >> total_vendas;
  
  cout << fixed << setprecision(2);
  
  double comissao = salario_fixo + (total_vendas*0.15);
    
  cout << "TOTAL = R$ " << comissao << endl;
  return 0;
}

File: BeeCrowd Python/Iniciante/1009.py
def main():
    nome = input()
    salario_fixo = float(input())
    vendas = float(input())
    
    comissao = vendas * 0.15
    montante = salario_fixo + comissao
    
    print(f"TOTAL = R$ {montante:.2f}")

main()
